### PR TITLE
fix(DB/gameobject): Darnassus Moonwell spell focus objects

### LINKED ISSUE #11325: [Darnassus][Tailoring] Temple of the Moon moonwell not a valid location for Felcloth to Mooncloth conversion.
### Current Behaviour

The moonwell & shrine at the "Temple of the Moon" in Darnassus will not allow you to make Mooncloth.

### Expected Blizzlike Behaviour

The moonwell & shrine at the "Temple of the Moon" in Darnassus is an eligible place to craft Mooncloth.

### Source

According to https://www.wowhead.com/item=14342/mooncloth#comments:id=253084 the fountain in Darnassus shoulf work as well, but it doesnt.

> Location of the Moonwells ("Moonwell"):
> Alliance:
> Temple of the Moon, Darnassus
> source: https://wow-battlenet-pl.translate.goog/wow/Postac/Profesje/557,Krawiectwo-Tailoring.html?_x_tr_sch=http&_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en-US&_x_tr_pto=wapp
> 
> Old-world locations are more numerous but less convenient for Horde tailors.
> [Teldrassil](http://www.wowwiki.com/Teldrassil): [Dolanaar](http://www.wowwiki.com/Dolanaar); [Pools of Arlithrien](http://www.wowwiki.com/Pools_of_Arlithrien); [The Oracle Glade](http://www.wowwiki.com/Oracle_Glade)
> [Darnassus](http://www.wowwiki.com/Darnassus): [Temple of the Moon](http://www.wowwiki.com/Temple_of_the_Moon)
> source: https://www.engadget.com/2007-05-11-insider-trader-new-mooncloth-on-monday.html?guccounter=1
> 
> [Teldrassil](https://wowwiki-archive.fandom.com/wiki/Teldrassil): Various settlements, including [Dolanaar](https://wowwiki-archive.fandom.com/wiki/Dolanaar)
> [Darnassus](https://wowwiki-archive.fandom.com/wiki/Darnassus): [Temple of the Moon](https://wowwiki-archive.fandom.com/wiki/Temple_of_the_Moon)
> source: https://wowwiki-archive.fandom.com/wiki/Moonwell
> 
> But now, you can convert 2 felcloth to 1 mooncloth for free. To do
> so, find a moonwell. If you are alliance, go to the one in Darnassus.
> The Darnassus moonwell is in the Temple of the Moon, straight south
> from the bank.
> source: https://www.ownedcore.com/forums/world-of-warcraft/world-of-warcraft-guides/12482-chezziks-1-375-tailoring-guide.html

[One more souce, this time referring to Classic WoW](https://www.reddit.com/r/classicwow/comments/ei9mqr/fyi_you_cant_use_the_moonwell_in_the_temple_of/)

 [Wowpedia Moonwell Talk Page](https://wowpedia.fandom.com/wiki/Talk:Moonwell):
> If I remember correctly, the "pool" in the temple is a moonwell. I think I used to teleport to darnassus, move some feet, create the cloth and teleport back to Ironforge, done within a few seconds. But that's some time ago, and i'm not completely sure. --[Ose](https://wowpedia.fandom.com/wiki/User:Ose) ([c](https://wowpedia.fandom.com/wiki/Special:Contributions/Ose) / t) 14:54, 3 March 2008 (UTC)
> Confirmed that the Darnassus Moonwell is a real Moonwell. [Jopernikus](https://wowpedia.fandom.com/wiki/User:Jopernikus) (talk) 03:17, 26 May 2008 (UTC)

### Steps to reproduce the problem

1. Travel to the Temple of the moon in Darnassus and go into the Moonwell.
2. Attempt to turn Felcloth into Mooncloth and you will get the error "requirement moonwell".

### Extra Notes

Originally Reported: https://github.com/chromiecraft/chromiecraft/issues/3215

Temple of the moon object ID
moonwell

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/e8d3292533c7e5d18af15d9bffabd581bacbadf9

### Operating system

Ubuntu 20.04

### Custom changes or Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)
- [lua-zone-debuff](https://github.com/55Honey/Acore_ZoneDebuff)

### PR DESCRIPTION
**Pull request by aradep**
## Changes Proposed:

Adds Moonwell spell focus gameobjects to the moonwell in the Temple of the Moon, Darnassus.

## Issues Addressed:

- Closes #11325 
- Closes chromiecraft/chromiecraft/issues/3215
## Tests Performed:

Tested in-game on latest build.

## How to Test the Changes:

1. Go to the Temple of the Moon in Darnassus.
2. Learn Mooncloth crafting and get some Felcloth. 
 .learn 26798
 .learn 18560
 .add 14256 2
3. Attempt to craft Mooncloth at various locations in and around the Moonwell.

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Lynethris** [comment]
Tested and work.

<details>

WoWScrnShot_041922_100851


WoWScrnShot_041922_100900

</details>

**Comment by Nefertumm** [comment]
Are these positions sniffed?

**Comment by aradep** [comment]
> Are these positions from sniffs?

Unfortunately not. For the other moonwells theres 1 focus object at the center, but this moonwell is a lot bigger and the statue in the middle takes up most of that useful space so I added 4.

**Comment by Nefertumm** [comment]
I can ask for someone to sniff it, so you can correct those values in your PR :D

**Comment by benjymansy123** [comment]
Submitted a sniff, don't know if WPP is up to date with TBCC though.

File: data/sql/updates/pending_db_world/rev_1650258156241061000.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1650258156241061000');
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1650258156241061000.sql:1]: delete this

**Comment by aradep** [comment]
Accidentally deleted the repo so I can't edit this anymore. Does the SQL file need the double dash -- on line 1 to work correctly?

**Comment by Kitzunu** [comment]
no

**Review by acidmanifesto — APPROVED** [review]

**Pull request merged**